alpha/declcfg/declcfg_to_model_test.go

```diff
@@ -280,6 +280,155 @@ func TestConvertToModel(t *testing.T) {
 				Bundles:  []Bundle{newTestBundle("foo", "0.1.0")},
 			},
 		},
+		{
+			name:      "Error/Deprecation/UnspecifiedPackage",
+			assertion: hasError(`package name must be set for deprecation item 0`),
```

Comment:
If the content of the error is valuable to callers, it should be exposed by a mechanism that they can query without resorting to stringly-typing - exported error types, `errors.Is()`, etc. Again, I know you're following the pattern here but this adds brittleness to the tests here